src/styles/stars.scss

```diff
@@ -0,0 +1,21 @@
+@import 'variables';
+
+.card__rate-stars {
+  display: flex;
+
+  &-star {
+    width: $star-size;
+    height: $star-size;
+
+    background: url(../images/star.svg) no-repeat center;
```

Comment:
Consider using a variable for the URL path of the star image to maintain consistency and make it easier to update if the path changes.